.taskcluster.yml

```diff
@@ -407,75 +407,43 @@ tasks:
   #           ineffassign .
   #     artifacts:
   #       - name: public/build/generic-worker-linux-armv6l
-  #         path: gopath1.10.3/bin/generic-worker
+  #         path: gopath1.10.4/bin/generic-worker
   #         expires: "{{ '2 weeks' | $fromNow }}"
   #         type: file
   #     mounts:
   #       - cacheName: generic-worker-checkout
-  #         directory: gopath1.10.3/src
+  #         directory: gopath1.10.4/src
   #       - content:
-  #           url: https://storage.googleapis.com/golang/go1.10.3.linux-armv6l.tar.gz
+  #           url: https://storage.googleapis.com/golang/go1.10.4.linux-armv6l.tar.gz
   #           sha256: d3df3fa3d153e81041af24f31a82f86a21cb7b92c1b5552fb621bad0320f06b6
-  #         directory: go1.10.3
+  #         directory: go1.10.4
   #         format: tar.gz
 
 
   ##########################################################
-  ################### Linux amd64 Build ####################
+  ##################### docker tests #######################
   ##########################################################
 
-  - provisionerId: "{{ taskcluster.docker.provisionerId }}"
-    workerType: "{{ taskcluster.docker.workerType }}"
+  - provisionerId: aws-provisioner-v1
+    workerType: github-worker
```

Comment:
This worker type executes its tasks inside a docker container, but we need to test generic-worker with docker engine, which will require launching docker. We therefore need to run the CI tasks as a native process on the host, rather than inside a docker container, so we'll need a generic-worker host running native engine on linux.

Reply:
I am giving docker-in-docker a try